ipfs-cluster-service/main.go

```diff
@@ -261,7 +261,82 @@ removal, upgrade state using this command, and restart every peer.
 					Action: func(c *cli.Context) error {
 						err := upgrade()
 						checkErr("upgrading state", err)
-						return err
+						return nil
+					},
+				},
+				{
+					Name:  "export",
+					Usage: "save the IPFS Cluster state to a human readable json file",
```

Comment:
is it human readable?

Reply:
yes, nvm

Reply:
Well I guess it depends on the human, definitely more so than msgpack :) if you prefer I can remove this wording

Reply:
it's good, at first it wasn't clear to me that we weren't exporting msgpack directly because of the Snapshot() function.